examples/child_window.rs

```diff
@@ -36,7 +36,7 @@ fn main() -> Result<(), impl std::error::Error> {
 
     let mut windows = HashMap::new();
 
-    let event_loop: EventLoop<()> = EventLoop::new();
+    let event_loop: EventLoop<()> = EventLoop::new().unwrap();
```

Comment:
I think we should use `?` in the examples instead, generally looks prettier.

Reply:
Well, we use `unwrap` for lots of other things as well in examples, making examples prettier is for the separate PR, I'd say.